daemon/README.md

```diff
@@ -236,6 +236,7 @@ following environment variables will control its creation:
 * `WEIGHT` is the of the OSD when it is added to the CRUSH map (default is `1.0`)
 * `JOURNAL` is the location of the journal (default is the `journal` file inside the OSD data directory)
 * `HOSTNAME` is the name of the host; it is used as a flag when adding the OSD to the CRUSH map
+* `OSD_JOURNAL<osd-id>` optional - providing for each OSD directory to be used will prepare and mount the supplied block device
```

Comment:
perhaps we can highlight that this only works for the osd directory scenario? or at least make it a bit clearer?